canonical-websites/www.ubuntu.com
# update /download/desktop to VBT

## Done

* converted /download/desktop to VBT
* added temp css to make brand button text white

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
* [test page](http://0.0.0.0:8001/download/desktop)
* [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/9.%20Download?preview=download-desktop.png)
* [demo](http://www.ubuntu.com-1625-download-desktop.demo.haus/download/desktop)


## Issue / Card

Fixes #1625
